### PR TITLE
Add initial HTC U12+ overlay

### PR DESCRIPTION
**Pull request by jeanrivera**
here is my attempt to create an overlay for HTC U12+

### CODE REVIEW

**Comment by phhusson** [comment]
Remove "fail" file
Rename "ime" folder to U12Plus or something actually meaning the actual device name, not codename

CI says:
F: overlay.mk: Keep entries sorted

Where are those values coming from? All values come from stock framework-res?

**Comment by jeanrivera** [comment]
yes the values comes from stock framework-res

**Comment by jeanrivera** [comment]
git remote add mygit https://github.com/jeanrivera/vendor_hardware_overlay.git
fatal: remote mygit already exists.
how do I fix that?

File: HTC/ime/Android.mk [hunk 1]
```diff
@@ -0,0 +1,8 @@
+LOCAL_PATH := $(call my-dir)
+include $(CLEAR_VARS)
+LOCAL_MODULE_TAGS := optional
+LOCAL_PACKAGE_NAME := treble-overlay-htc-ime
```
Comment (TadiT7) [inline HTC/ime/Android.mk]:
```suggestion
LOCAL_PACKAGE_NAME := treble-overlay-htc-u12plus
```

File: HTC/ime/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.htc.ime"
```
Comment (TadiT7) [inline HTC/ime/AndroidManifest.xml]:
```suggestion
        package="me.phh.treble.overlay.htc.u12plus"
```

File: overlay.mk [hunk 3]
```diff
@@ -65,6 +65,7 @@ PRODUCT_PACKAGES += \
 	treble-overlay-xiaomi-redminote6pro-systemui \
 	treble-overlay-xiaomi-redminote7 \
 	treble-overlay-xiaomi-redmis2 \
+	treble-overlay-htc-ime \
```
Comment (TadiT7) [inline overlay.mk]:
```suggestion
	treble-overlay-htc-u12plus \
```

**Comment by TadiT7** [comment]
> git remote add mygit https://github.com/jeanrivera/vendor_hardware_overlay.git
> fatal: remote mygit already exists.
> how do I fix that?

you don't need to add the remote twice, just push again

**Comment by TadiT7** [comment]
Delete unused comments in config.xml

File: HTC/U12Plus/AndroidManifest.xml [hunk 4]
```diff
@@ -6,5 +6,5 @@
                 android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
 		android:requiredSystemPropertyValue="+htc/ime*"
 		android:priority="97"
-		android:isStatic="true" />
+		android:isStatic="false" />
```
Comment (TadiT7) [inline HTC/U12Plus/AndroidManifest.xml]: Why? We need it to be static.

Comment (jeanrivera) [reply HTC/U12Plus/AndroidManifest.xml]: Dil3ma told me to make it dynamic I don't know why I just try anything I being told to try to make it work

Comment (TadiT7) [reply HTC/U12Plus/AndroidManifest.xml]: It just needs to be active. We'll try to handle the rest then, but first you need to clean up this overlay.

Comment (jeanrivera) [reply HTC/U12Plus/AndroidManifest.xml]: He sent me one that was active after the command he told me he said he will PR to phh

**Comment by Manish4586** [comment]
> git remote add mygit https://github.com/jeanrivera/vendor_hardware_overlay.git
> fatal: remote mygit already exists.
> how do I fix that?

You don't need to add remove twice

**Comment by phhusson** [comment]
merged in c17fd05

**Comment by phhusson** [comment]
(as static overlay ofc)

**Comment by jeanrivera** [comment]
Merged thanks!!

**Pull request closed without merging**